apps/src/templates/rubrics/introjs.scss

```diff
@@ -445,4 +445,5 @@ tr.introjs-showElement > th {
   z-index: 1;
   opacity: 0;
 }
-/*# sourceMappingURL=introjs.css.map */
+
+/* eslint-disable import/order */
```

Comment:
I don't think this belongs here :smile: 

Also looks like the unintended removal above is still present in the changeset